Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -22,8 +34,23 @@ YUI.add('ez-searchplugin-tests', function (Y) {
             this.capi = new Mock();
             this.contentService = new Mock();
             this.viewName = 'REST-View-Name';
-            this.query = {body: {ViewInput: {ContentQuery: {}}}};
-
+            this.query = new Y.Mock();
+            this.sortClauses = {};
+            Mock.expect(this.query, {
+                method: 'setLimitAndOffset',
+                args: [Mock.Value.Number, Mock.Value.Number],
+            });
+            Mock.expect(this.query, {
+                method: 'setSortClauses',
+                args: [Mock.Value.Object],
```

Comment:
instead of having a `run` function with an assertion, you can directly put `this.sortClauses` here